packages/cli/lib/index.ts

```diff
@@ -43,15 +45,26 @@ class NangoCommand extends Command {
         const cmd = new Command(name);
         cmd.option('--auto-confirm', 'Auto confirm yes to all prompts.', false);
         cmd.option('--debug', 'Run cli in debug mode, outputting verbose logs.', false);
+        // Defining the option with --no- prefix makes it true by default.
+        // The option name in the code will be 'interactive'.
+        // Passing --no-interactive will set it to false.
+        cmd.option('--no-interactive', 'Disable interactive prompts for missing arguments.');
+
         cmd.hook('preAction', async function (this: Command, actionCommand: Command) {
-            const { debug } = actionCommand.opts<GlobalOptions>();
-            printDebug('Debug mode enabled', debug);
-            if (debug && fs.existsSync('.env')) {
-                printDebug('.env file detected and loaded', debug);
+            const opts = actionCommand.opts<GlobalOptions>();
+
+            // opts.interactive is true by default (from the option default), or false if --no-interactive is passed.
+            // We also disable it if we are in a CI environment.
+            opts.interactive = opts.interactive && !isCI;
```

Comment:
When would this be used in a CI environment? Is `CI` an industry standard env var that other tools rely on to alter their behavior?

Reply:
Some customers use the CLI today on CI envs to test and deploy nango. If we force them to adopt the `--no-interactive` flag we would break their CI pipelines. At the same time, we want interactive mode to be the default use case for people using the CLI on the terminal, so this gets both things done. A warning for CI envs that do not provide the `--no-interactive` flag might be in order though. Wdyt?

Reply:
Yup I think a warning would be great.